src/Components/EmailNotificationForm/EmailNotificationForm.js

```diff
@@ -0,0 +1,271 @@
+import { Component } from "react";
+import './EmailNotificationForm.css';
+
+class EmailNotificationForm extends Component {
+  constructor() {
+    super();
+    this.state = {
+      first_name: '',
+      last_name: '',
+      state: '',
```

Comment:
Did you come across any hiccups using 'state' as a key in 'state'?

Reply:
I did not have any issues using 'this.state.state', but I've refactored this property to state_name.